packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.scss

```diff
@@ -32,198 +32,195 @@ $ms-checkbox-highContrast-iconBoxSize: 16px;
   }
 }
 
-:global {
-
-  .ms-Checkbox {
-    box-sizing: border-box;
-    color: $ms-color-neutralPrimary;
-    font-family: $ms-font-family-base;
-    font-size: $ms-font-size-m;
-    font-weight: $ms-font-weight-regular;
-    min-height: 36px;
-    position: relative;
-
-    .ms-Label {
-      font-size: $ms-font-size-m;
-      @include padding(0, 0, 0, 26px);
-      display: inline-block;
-    }
-  }
+.root {
+  box-sizing: border-box;
+  color: $ms-color-neutralPrimary;
+  font-family: $ms-font-family-base;
+  font-size: $ms-font-size-m;
+  font-weight: $ms-font-weight-regular;
+  min-height: 36px;
+  position: relative;
+}
 
-  // The hidden input
-  .ms-Checkbox-input {
-    position: absolute;
-    opacity: 0;
-    top: 8px
-  }
+.textLabel {
```

Comment:
Checkbox has a label for the checkbox, and a label that displays the text. I also was confused about this, but i'm not changing the DOM to make it more clear.